sale_order_warn_message/tests/test_sale_order_warn_message.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrderWarnMessage(TransactionCase):
```

Comment:
same remark as https://github.com/OCA/social/pull/479#discussion_r509183882
Not blocking, but if you merge pls consider opening anothe PR to fix this :pray: